src/azul/__init__.py

```diff
@@ -1141,6 +1141,12 @@ def cloudtrail_s3_bucket_region(self) -> str:
     def cloudtrail_trail_region(self) -> str:
         return self.environ['azul_cloudtrail_trail_region']
 
+    @property
+    def azul_notification_email(self) -> str:
+        return self.environ['AZUL_NOTIFICATION_EMAIL']
+
+    sns_topic_base_name = 'topic'
```

Comment:
I only see this used as a parameter to qualified_resource_name so that call can be extracted into a property.

Additionally, we typically don't encode the resource type in a resource name. So

```suggestion
    sns_topic_base_name = 'monitoring'
```